spec/views/catalog/show.html.erb_spec.rb

```diff
@@ -66,14 +66,14 @@
   describe 'the location for physical holdings', js: true do
     context 'if physical holding information is recorded in the entry' do
       it 'is not rendered' do
-        visit 'catalog/857469'
-        expect(page).not_to have_selector('#doc_857469 > dl > dt.blacklight-holdings_1display')
+        visit 'catalog/9960108133506421'
```

Comment:
Question: Why is this not 998574693506421?

Reply:
None of the ids have the class. I was switching between the ids to make sure that it is not rendered. I will open an issue about this blacklight class. I think it might be because now we display only the holding information and not the item but I dont know yet.

Reply:
https://github.com/pulibrary/orangelight/issues/2663